analysis/1/top5_country_suicide_rate_comparison_by_sex.ipynb

```diff
@@ -95,7 +95,7 @@
    ],
    "source": [
     "## Get the countries names of top 5 highest suicide rate\n",
-    "print(top_10_suicide_rate_bothsex['Country'].to_list())\n",
+    "print(top_5_suicide_rate_bothsex['Country'].to_list())\n",
     "top_10_country_names = top_5_suicide_rate_bothsex['Country'].to_list()\n",
     "## Filtering out Male/Female records belongs to the hightest country.\n",
     "final = suicide_rate_separatesex[suicide_rate_separatesex['Country'].isin(top_10_country_names)]\n",
```

Comment:
prefer to use a more readable variable name, instead of `final` ...

Reply:
Thanks, Irfan.